tests/tpg361.py

```diff
@@ -0,0 +1,122 @@
+import unittest
+
+from common_tests.tpgx6x import TpgBase, ErrorFlags, ErrorStrings
+from utils.ioc_launcher import get_default_ioc_dir
+from utils.test_modes import TestModes
+from parameterized import parameterized
+
+
+DEVICE_PREFIX = "TPG36X_01"
+
+
+IOCS = [
+    {
+        "name": DEVICE_PREFIX,
+        "directory": get_default_ioc_dir("TPG36X"),
+        "emulator": "tpgx6x",
+        "lewis_protocol": "tpg361",
+        "macros": {
+            "IS361": "Y"
+        }
+    },
+]
+
+
+TEST_MODES = [TestModes.RECSIM, TestModes.DEVSIM]
+
+
+class Tpg361Tests(TpgBase, unittest.TestCase):
+    def get_prefix(self):
+        return DEVICE_PREFIX
+
+    @parameterized.expand(("Pressure_{}".format(i), i) for i in range(1, 3))
+    def test_GIVEN_pressure_set_WHEN_read_THEN_pressure_is_as_expected(self, _, channel):
+        if channel == 1:
```

Comment:
We're parametrizing the whole test for 3 channels but then only actually testing one of them. I think we can remove the parameterized decorator and the `if`-case in this test.

Comment repeats for all similar cases